lib/diagnostic.rb

```diff
@@ -71,18 +72,18 @@
 
 # What value will be returned. Write your response as a Ruby string.
 
-Diagnostic.flow_control = 'your response here'
+Diagnostic.flow_control = 'Just your average billionaire'
 
```

Comment:
it would actually be `'The Dark Knight'`
we are checking if batman is true, and it is since we defined it above!